packages/run-protocol/src/runStake/runStake.js

```diff
@@ -191,7 +195,7 @@ export const start = async (
   const publicFacet = augmentPublicFacet(
     Far('runStake public', {
       makeLoanInvitation: () =>
-        zcf.makeInvitation(offerHandler, 'make RUNstake'),
+        zcf.makeInvitation(handleLoanOffer, 'make RUNstake'),
```

Comment:
Purposeful violation? If intentional, I'm not objecting. But worth at least a comment to explain why.

Reply:
Purposeful violation of the string? yeah, for comments above. 

I believe the method names are consistent. (`Loan` as key)